Packages/com.vrchat.ClientSim/Runtime/System/ClientSimPlayerManager.cs

```diff
@@ -685,6 +685,63 @@ public static string[] GetAvailableLanguages()
         }
 
         #endregion
+        
+        #region Player Scaling
+        
+        private const float AVATAR_MIN_EYE_HEIGHT_DEFAULT = 0.2f;
+        private const float AVATAR_MAX_EYE_HEIGHT_DEFAULT = 5f;
+        private const float AVATAR_EYE_HEIGHT_DEFAULT = 1.6f;
```

Comment:
The default in ClientSim is actually 1.9.

But I think we can remove this because of the suggestion below.
```suggestion

```